ingest-mapper/src/main/scala/uk/gov/nationalarchives/ingestmapper/MetadataService.scala

```diff
@@ -46,7 +55,7 @@ class MetadataService(s3: DAS3Client[IO]) {
             .getOrElse(departmentId)
           val parentPaths = getParentPaths(json)
           Stream.emits {
-            json.arr.toList.map { metadataEntry =>
+            val jsonArr = json.arr.toList.map { metadataEntry =>
```

Comment:
Maybe `updatedJson` or  `updatedJsonObj`?

Reply:
`updatedJson` is ok.